lms/djangoapps/discussion/config/settings.py

```diff
@@ -19,4 +19,4 @@
 
 def is_forum_daily_digest_enabled():
     """Returns whether forum notification features should be visible"""
-    return settings.FEATURES.get('ENABLE_FORUM_DAILY_DIGEST')
+    return settings.FEATURES.get('ENABLE_FORUM_DAILY_DIGEST', True)
```

Comment:
Note that the default value is duplicated both here in the code and also in the annotation above.  I'm not sure if there's any better way to DRY this though - without over-complicating.

Reply:
Let me know if you have any suggestions. I think the annotation extraction will get complicated if it is using annotations + looking at code, but that is possible. It is also possible for the linter, which is looking at both, to try to warn about mismatches. But this might require a wrapper around the setting like WaffleFlag wraps waffle.

Anyway, all of this seems outside the scope of this PR. Do you agree?